src/spaceone/inventory/manager/workload/deployment_manager.py

```diff
@@ -180,11 +179,19 @@ def _get_pods(self, list_pods, list_replicasets, deployment_uid):
         pods_for_deployment = []
         for pod in list_pods:
             raw_pod = pod.to_dict()
+
+            if raw_pod.get("metadata").get("owner_references") is None:
+                continue
+
             pod_owner_reference = raw_pod.get("metadata", {}).get(
                 "owner_references", []
             )[0]
             for replicaset in list_replicasets:
                 raw_replicaset = replicaset.to_dict()
+
+                if raw_replicaset.get("metadata").get("owner_references") is None:
+                    continue
+
                 replicaset_owner_reference = raw_replicaset.get("metadata", {}).get(
                     "owner_references", []
                 )[0]
```

Comment:
Index access like [0] is hard to know what it means when you first look at the code.
I don't know if this applies to your case, but it's something to consider.

[TO-BE]
```python
a, b = raw_replicaset.get("metadata", {}).get(
                     "owner_references", []
                 )
```
When we use `a` variable, it is clear to figure out what `b` is also.

Reply:
Why did you recommend tuple types such as `a, b` for list return values from `.get("owner_references", [])`?

Reply:
If the list types are all the same object, using [0] to mean accessing the first index seems fine.

However, if the objects entering the list type are different, 
I thought it would be better to clearly express what y means regardless of the list length by expressing it like `x, *y = list type`.

Taking a closer look at the example above

```python
first_owner_ref, *the_others = raw_replicaset.get("metadata", {}).get(
                     "owner_references", []
                 )
```

I thought it would be clear what the rest meant if I gave the_others a proper variable name.